packages/gestalt-datepicker/rollup.config.js

```diff
@@ -1,5 +1,5 @@
-// eslint-disable-next-line flowtype/require-valid-file-annotation
-import plugins from '../gestalt-core/build.js'; // eslint-disable-line import/no-relative-parent-imports
+// eslint-disable-next-line flowtype/require-valid-file-annotation, import/no-relative-parent-imports, import/no-relative-packages
```

Comment:
Not sure if this is something we actually need to fix:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-relative-packages.md